src/PlayFabBuddy.Cli/Program.cs

```diff
@@ -32,6 +32,7 @@ public async static Task<int> Main(string[] args)
             {
                 players.AddCommand<CreateNewPlayersCommand>("create");
                 players.AddCommand<DeleteAllPlayersCommand>("delete");
+                players.AddCommand<DeletePlayersBySegmentCommand>("delete-by-segment");
```

Comment:
Do we agree that this needs to be a new command "players delete-by-segment" or should it be a parameter like "players delete --by-segment segment name.

I would prefer the parameter version as dev.

Reply:
Generally agree - but the otehr delete command does something completely different. Shall we change the existing "delete from JSON" as well? I would be much in favour. e.g `--from-json`?

Reply:
Okay let's fix this in #53